### ISSUE TITLE
Upload package to DevOps with new version

### ISSUE THREAD

**Issue by readyforchaos** [RaaLabs/TSIClient#4]

**Comment by readyforchaos** [comment]
This is blocked until @KristofferBerge can take a look at it.

**I have also deleted TSIClient from PyPi as there was a conflict it seems:**

```
C:\Users\gill\Desktop\localPackage>twine upload --repository-url "https://pkgs.dev.azure.com/raalabs/_packaging/raalabs/pypi/upload" dist/*
Enter your username: raalabs
Enter your password:
Uploading distributions to https://pkgs.dev.azure.com/raalabs/_packaging/raalabs/pypi/upload
Uploading TSIClient-0.7-py3-none-any.whl
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 14.3k/14.3k [00:00<00:00, 27.5kB/s]
NOTE: Try --verbose to see response content.
HTTPError: 409 Client Error: Conflict - 'TSIClient 0.7' cannot be published to the feed because it exists in at least one of the feed's upstream sources. Publishing this copy would prevent you from using 'TSIClient 0.7' from 'PyPI'. For more information, see https://go.microsoft.com/fwlink/?linkid=864880 (DevOps Activity ID: 4946B9D2-EA30-4649-A1DB-C9887FFD98C4) for url: https://pkgs.dev.azure.com/raalabs/_packaging/raalabs/pypi/upload
```

I thought that deleting the pypi release would make it available for Azure DevOps but it didn't fix the problem. I can upload it to PyPi later if this doesn't work but lets try to make it work on Azure DevOps first.

TLDR:
Will have to find a way of uploading it to Azure DevOps before TSIClient is useable.
Until that, use the TSIClient-code inline in your code if you need to extract data from TSI.
fyi: @SiriOvregard @EmilRamsvik

**Comment by readyforchaos** [comment]
> "saved packages remain in the feed even if the upstream source is disabled or removed"
From: https://docs.microsoft.com/nb-no/azure/devops/artifacts/concepts/upstream-sources?view=azure-devops#determining-the-package-to-use-search-order

No wonder why deleting in on pypi didn't work.

**Comment by readyforchaos** [comment]
Managed to upload it to Azure DevOps!
I had to delete all upstreams, but this means that you cannot install the .whl from pypi because of immutability since I deleted it earlier.

A possible fix might be to rename the whole package and start the uploading process from there so contain upstream as well.

**Comment by rafaelschlatter** [comment]
> TLDR:
> Will have to find a way of uploading it to Azure DevOps before TSIClient is useable.
> Until that, use the TSIClient-code inline in your code if you need to extract data from TSI.
> fyi: @SiriOvregard @EmilRamsvik

You can also install this from GitHub directly, like this:
`pip install git+https://github.com/RaaLabs/TSIClient.git`

**Comment by rafaelschlatter** [comment]
This seems to solved. Closing this.
